components/template/TemplateCard.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="template">
+    <NuxtLink :to="{ name: 'templates-slug', params: { slug: template.slug } }">
+      <div v-if="loading" class="image-skeleton"></div>
+      <img
+        v-if="template.previewImageUrl"
+        class="img-fluid pointer template-preview__img"
+        :class="{ hidden: loading }"
+        :src="template.previewImageUrl"
+        :alt="template.name"
+        @load="loading = false"
+      />
+      <img
+        v-else
+        class="img-fluid pointer template-preview__img"
+        :class="{ hidden: loading }"
```

Comment:
The condition should be inverted for placeholder right?

Reply:
No, we are not showing placeholder image untill the actual image loads. we are showing a skeleton when image is loading